Prepend 'clawpack' to pyclaw autodoc paths.

ketch opened the pull request:
This fixes everything brought up by @MathematicianVogt  in https://github.com/clawpack/clawpack/issues/119, as well as all other instances I could find.  The problem was that back when we put pyclaw under clawpack, we didn't fix all the autodoc paths to start with clawpack.

rjleveque commented:
@ketch: Are you going to rebuild the docs or do you want me to?

Recall it's complicated now because of the versioning.  There are some notes at
  http://www.clawpack.org/howto_doc.html
I can tackle it if you want.

ketch commented:
I seem to recall that there are some glitches with the PyClaw gallery that I need to fix.  I will see if I can rebuild everything.

mandli commented:
I just noticed again that the docs are not fixed yet on clawpack.org.  @ketch and @rjleveque do you guys need help with this?

ketch commented:
It looks like @rjleveque rebuilt the docs so these changes were incorporated (thanks!).  I just fixed a couple more broken links to the old location of the pyclaw gallery (and removed that old directory).

mandli commented:
Do we need a new build and PR then?

rjleveque commented:
It looks like @ketch pushed a new version of the docs, but some things are still broken, e.g. http://www.clawpack.org/pyclaw/evolve/limiters.html
